hack/e2e_dev.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+: "${KUBECONFIG:?}"
+
+ROOT_DIR=$(dirname "${BASH_SOURCE[0]}")/..
+source "${ROOT_DIR}/hack/common.sh"
+source "${ROOT_DIR}/hack/lib/tests.sh"
+
+function cleanup() {
+    exit_status=$?
+
+#    echo "Removing namespaces with the 'name=metering-testing-ns' label"
+#    kubectl delete ns -l "name=metering-testing-ns" || true
```

Comment:
This is just a copy of `hack/e2e.sh` that avoids deleting the project -- for dev trialing of a new test.

There is a also a conditional on Teardown section set to `false` here: https://github.com/operator-framework/operator-metering/pull/1064/commits/5174f648ed72ef82ffead8eeea8e26e4fe7edd99#diff-e8664074431f2653c7da0fc9019a1935R482